src/app/view/components/dot-relationship-tree/dot-relationship-tree.component.html

```diff
@@ -0,0 +1,30 @@
+<ul class="dot-tree">
+    <li>
+        <div class="dot-tree__content">
+            <dot-icon-button
+                icon="wysiwyg"
```

Comment:
The icons needs to be dynamic. We have [this service](https://github.com/dotCMS/core-web/blob/issue-18706-dot-relationship-field-component/src/app/api/services/dot-content-types-info/dot-content-types-info.service.ts#L9) we use to pull the icon. Check how we use it in other places.